backend/api/src/handlers/friendships/get_friendship_status.rs

```diff
@@ -0,0 +1,42 @@
+use actix_web::{web, HttpResponse};
+use arcadia_common::error::Result;
+use arcadia_storage::{connection_pool::ConnectionPool, models::friendship::FriendshipStatus};
+use utoipa::OpenApi;
+
+use crate::handlers::UserId;
+
+#[utoipa::path(
+    get,
+    path = "/api/friendships/status/{user_id}",
+    params(
+        ("user_id" = i64, Path, description = "ID of the user to check friendship status with")
+    ),
+    responses(
+        (status = 200, description = "Friendship status between users", body = FriendshipStatus),
+        (status = 401, description = "Unauthorized"),
+        (status = 404, description = "User not found")
+    ),
+    security(
```

Comment:
add this for the time being 

```
    security(
      ("http" = ["Bearer"])
    ),
```

I'm working on the PR that will know how to interpret this